testing/fetcher.ts

```diff
@@ -270,7 +270,7 @@ export class AuthenticatingFetcher implements Fetcher {
     if (!this._credentials) {
       throw new Error(
         `${this._authDef.type} authentication is required for this pack, but no local credentials were found. ` +
-          'Run "coda auth path/to/pack/manifest to set up credentials."',
```

Comment:
I think my VSCode is handling these continued lines somewhat differently, but it lints fine. Is there something I can do to bring this in line with the rest of the team?